test/unit/org/apache/cassandra/cql3/ast/Element.java

```diff
@@ -22,17 +22,24 @@
 
 public interface Element
 {
-    void toCQL(StringBuilder sb, int indent);
+    void toCQL(StringBuilder sb, CQLFormatter formatter);
 
     default void toCQL(StringBuilder sb)
     {
-        toCQL(sb, 0);
+        toCQL(sb, CQLFormatter.None.instance);
```

Comment:
before we would pretty print by default, but to avoid the extra allocation switched to `None`... I kinda feel anyone who cares should always define just to make sure this default can change over time.